Classes/LinphoneManager.m

```diff
@@ -1910,15 +1910,12 @@ - (void)call:(NSString *)address displayName:(NSString *)displayName transfer:(B
     if ([[LinphoneManager instance] lpConfigBoolForKey:@"rtt"]) {
         linphone_call_params_enable_realtime_text(lcallParams, linphone_core_realtime_text_enabled(theLinphoneCore));
     }
-    // VTCSecure add user location when emergency number is dialled.
-    NSString *emergency = [[LinphoneManager instance] lpConfigStringForKey:@"emergency_username" forSection:@"vtcsecure"];
-    if (emergency != nil && ([address hasPrefix:emergency] || [address hasPrefix:[@"sip:" stringByAppendingString:emergency]])) {
-        NSString *locationString;
-        if (![[LinphoneLocationManager sharedManager] isAuthorized:TRUE]) locationString = NSLocalizedString(@"not authorized by user",nil);
-        else if (![[LinphoneLocationManager sharedManager] locationPlausible]) locationString =  NSLocalizedString(@"user location could not be established",nil);
-        else locationString  =  [[LinphoneLocationManager sharedManager] currentLocationAsText];
-        linphone_call_params_add_custom_header(lcallParams,"userLocation",[locationString cStringUsingEncoding:[NSString defaultCStringEncoding]]);
+    
+    if ([address isEqualToString:@"911"]) {
```

Comment:
If you find there is a sense of having address configuration it's not problem to leave that instead of hardcoding to 911 ?